config/application.rb

```diff
@@ -11,7 +11,6 @@ module GramGram
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
-    Dotenv::Railtie.load
```

Comment:
@TimKJones I had to remove this line because it doesn't work in production. Do you need it earlier?

Reply:
The only thing that could potentially break is this line in `payment.html.erb` if ENV isn't loaded yet:
```
var stripe = Stripe("<%=ENV['STRIPE_PUBLISHABLE_KEY']%>");
```
Wanna see if the form loads at `/payment` in production?